tools/dependency/release_dates.py

```diff
@@ -40,32 +42,58 @@ def verify_and_print_latest_release(dep, repo, metadata_version, release_date):
         latest_release = repo.get_latest_release()
         if latest_release.created_at > release_date and latest_release.tag_name != metadata_version:
             print(
-                f'*WARNING* {dep} has a newer release than {metadata_version}@<{release_date}>: '
-                f'{latest_release.tag_name}@<{latest_release.created_at}>')
-    except github.UnknownObjectException:
+                Fore.YELLOW
+                + f'*WARNING* {dep} has a newer release than {metadata_version}@<{release_date}>: '
+                f'{latest_release.tag_name}@<{latest_release.created_at}>' + Style.RESET_ALL)
+    except github.GithubException as e:
```

Comment:
as this failure is silently ignored - can we add a comment explaining why it might be triggered and is safe to ignore